SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -476,9 +476,11 @@ void CDiscAdjSolver::SetAdj_ObjFunc(CGeometry *geometry, CConfig *config) {
 
 void CDiscAdjSolver::ExtractAdjoint_Solution(CGeometry *geometry, CConfig *config){
 
-  bool time_n1_needed = config->GetTime_Marching() == DT_STEPPING_2ND;
-  bool time_n_needed = (config->GetTime_Marching() == DT_STEPPING_1ST) || time_n1_needed;
-  bool multizone = config->GetMultizone_Problem();
+  const bool time_n1_needed = config->GetTime_Marching() == DT_STEPPING_2ND;
+  const bool time_n_needed = (config->GetTime_Marching() == DT_STEPPING_1ST) || time_n1_needed;
+  const bool multizone = config->GetMultizone_Problem();
+
+  const su2double relax = (config->GetInnerIter()==0)? 1.0 : config->GetRelaxation_Factor_AdjFlow();
```

Comment:
Perhaps we should remove the term "Flow" from this parameter if it will just be a general adjoint relaxation factor for any physical model.. 

I also assume we can assign this in each config separately for multiphysics calculations so that each solver does not need to apply the same value?

Reply:
I agree with that, but this goes back to my question during the last Dev meeting, do we want to maintain backwards compatibility in minor releases?

Reply:
I would love to maintain backward compatability, but I fear it is not really possible. Unless we kind of remember before every major release what config options we wanted to change... maybe it is enough if we throw an error that options have been renamed/removed. We already do that for some in CConfig::SetConfig_Parsing (which btw can be cleaned up anyway).

However in that particular case we first have to decide whether we need a different value for each individual solver I assume ..

Reply:
It is in part for this purpose that I opened #942, so that we can list options we think could be better etc.

Reply:
I vote one size fits all, otherwise we need to get into the whole `if KindSolver == ...`

Reply:
We could try to be much more systematic about it and do proper deprecation, i.e., find a way to support both a new option and the old one (use version numbers internally?), but mark the old one deprecated and remove in a future release (along with other similarly marked options). A little more effort, but it would achieve what we want

Reply:
I went with @talbring 's suggestion.